### PR TITLE
Completed JS Forms

### PR DESCRIPTION
**Pull request by macyhongg**
## Form Events
Kapture 2022-06-27 at 19 53 40
## Form Values
Kapture 2022-06-27 at 19 59 54

### CODE REVIEW

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var $form = document.getElementById('contact-form');
+$form.addEventListener('submit', submit)
```
Comment (macyhongg) [inline javascript-forms/form-values.js:2]: This should go at the very end

File: javascript-forms/form-events.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of', event.target.name, ":", event.target.value)
+}
+
+var $name = document.querySelector("input[name='name']");
```
Comment (macyhongg) [inline javascript-forms/form-events.js:15]: Whenever possible, use id instead

File: javascript-forms/form-events.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// var $name = document.forms.name
+//why didn't` this method work?
```
Comment (macyhongg) [inline javascript-forms/form-events.js:17]: document.forms[0].name
^^ add index

**Pull request merged**